### PR TITLE
feat: staking zApp breakpoints implementation

### PR DESCRIPTION
**Pull request by domw30**
[Associated Notion Card](https://wilderworld.atlassian.net/jira/software/c/projects/ZAP/boards/34?modal=detail&selectedIssue=ZAP-163)

-----

## 1. Pull request checklist

- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
- Feature

## 3. What is the old behaviour?
Breakpoints not implemented.

## 4. What is the new behaviour?
Breakpoints implements and style additions/refactoring.

https://user-images.githubusercontent.com/39112648/233448656-8fe5cd97-764b-4055-a95e-b101abb01fd2.mov

### CODE REVIEW

**Comment by domw30** [comment]
A couple of text wrapping issues that need to be handled in zUI or given design instruction. Large numbers (amount staked for example) are an issue and need truncating or another solution from design. These should be handled in another PR with the correct scope.

**Comment by domw30** [comment]
Looks like linting/prettier has re-formatted package-lock.json and package.json bringing the additions up

File: src/features/view-pools/PoolTable.module.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	display: flex;
+	flex-direction: column;
+	gap: 1.625rem;
+	position: relative;
+
+	.Grid {
+		grid-template-columns: 1fr;
+	}
+
+	.Table {
```
[the remark this answers is not in the document]
Comment (colbr) [reply src/features/view-pools/PoolTable.module.scss:13]: nit: could use `ResizeObserver` and toggle grid view based on container width. Not the ideal long term solution, but I'm thinking hiding the table might still be too resource intensive for people w slower machines. Little bit nitpicky potentially so no need to switch it!

**Review by colbr — APPROVED** [review]
Tidy ☺️

**Pull request merged**